src/main/java/org/jongo/marshall/jackson/JacksonMapper.java

```diff
@@ -73,7 +72,7 @@ public Mapper build() {
                 queryFactory = new BsonQueryFactory(jacksonEngine);
             }
             if (objectIdUpdater == null) {
-                objectIdUpdater = new ReflectiveObjectIdUpdater(new JacksonIdFieldSelector());
```

Comment:
Should the classes that are no longer being used be totally removed from the project, or are deprecation warnings the way you want to go?